### DISCUSSION TITLE
What's the meaning behind the retention times heatmap?

### DISCUSSION THREAD

**Discussion by Masonwei32** [vdemichev/DiaNN#1003] [Q&A]
Hello! I have a question about the retention times heatmap in output report part. Sometimes, I saw a bad correlation between RT and library iRT or predicted RT. What's the meaning? How to correct this problem? Any comment will be highly appreciated.

**Comment by vdemichev** [comment]
Can you please share more details on the problem, i.e. what exactly is being analysed, how the plot looks like, how the log looks like?

**Reply by Masonwei32** [reply]
Thank you, vdemichev! I attached a slide. Basically, I ran two files with the same range coverage and dynamic algorithm. Here, the range coverage was consistent with the DIA setting. I found out there appeared a much difference in retention times heatmap. I would concern that the result of bad correlation was not convincing. If so, how can I modify parameters to make it a good fit? Many thanks! 
[04302024_DIANN_Retention_times_heatmap.pptx](https://github.com/vdemichev/DiaNN/files/15164426/04302024_DIANN_Retention_times_heatmap.pptx)

**Reply by vdemichev** [reply]
Apologies for the late response, I have missed your reply.
Can be something like this if you are analysing with modification on which DIA-NN's predictor was not trained on. Is this the case? How does the log look like?
